### PR TITLE
Docs: biometric integration article

### PR DESCRIPTION
**Pull request by karthikeyan5**
This PR adds an article for biometric integration and steps to setup auto attendance.

- The Article
article_biometric_erpnext

- Auto Attendance Setup
auto_attendance_setup

### CODE REVIEW

File: erpnext_com/www/docs/user/manual/en/setting-up/articles/integrating-erpnext-with-biometric-attendance-devices.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+Attendance can then be marked based on the Employee Checkin records and the shift [Auto Attendance Settings](/docs/user/manual/en/human-resources/shift-management#25-auto-attendance-settings) of the employee by using [Auto Attendance](/docs/user/manual/en/human-resources/auto-attendance)
+
+Hence, integrating your Biometric Device --or any access control system that collects IN/OUT--, can be done by the following steps:
+
+  1. [Setting up Auto Attendance to mark attendance from the Employee Checkin](#1-setting-up-auto-attendance-to-mark-attendance-from-the-employee-checkin)
+  1. [Populating the Biometric Punch Logs into ERPNext's Employee Checkin](#2-populating-the-biometric-punch-logs-into-erpnexts-employee-checkin)
+
+## 1. Setting up Auto Attendance to mark attendance from the Employee Checkin
+
+Before you import employee's Check-in and Check-out logs into your ERPNext system. You would have to set up the Employees and their shifts to be able to generate attendance using the Auto Attendance feature in ERPNext.
```
Comment (michellealva) [inline erpnext_com/www/docs/user/manual/en/setting-up/articles/integrating-erpnext-with-biometric-attendance-devices.md]: Before you import employee's Check-in and Check-out logs into your ERPNext system, you would have to set up the employees and their shifts to be able to generate attendance using the Auto Attendance feature in ERPNext.

File: erpnext_com/www/docs/user/manual/en/setting-up/articles/integrating-erpnext-with-biometric-attendance-devices.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+Please refer to the following link to setup Auto Attendance: [Setup Auto Attendance](/docs/user/manual/en/human-resources/auto-attendance#steps-to-setup-auto-attendance)
```
Comment (michellealva) [inline erpnext_com/www/docs/user/manual/en/setting-up/articles/integrating-erpnext-with-biometric-attendance-devices.md]: Please refer to the following link to set up Auto Attendance:

Comment (michellealva) [reply erpnext_com/www/docs/user/manual/en/setting-up/articles/integrating-erpnext-with-biometric-attendance-devices.md]: even the link label should be Set Up Auto Attendance

File: erpnext_com/www/docs/user/manual/en/setting-up/articles/integrating-erpnext-with-biometric-attendance-devices.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+Once you have set up the Employee and assigned shifts to employees, you are now ready to proceed to the next step.
```
Comment (michellealva) [inline erpnext_com/www/docs/user/manual/en/setting-up/articles/integrating-erpnext-with-biometric-attendance-devices.md]: Once you have set up the Employee and assigned shifts to the employees, you are now ready to proceed to the next step.

File: erpnext_com/www/docs/user/manual/en/setting-up/articles/integrating-erpnext-with-biometric-attendance-devices.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+## 2. Populating the Biometric Punch Logs into ERPNext's Employee Checkin
+Depending on your biometric system and its features, there can be a lot of ways you can import the Punch logs into ERPNext:
+
+1. Use ERPNext's Data Import Tool:
+    - The simplest possible solution(in terms of implementation complexity) would be to generate an Excel/CSV of the Check-in/Check-out and use the built-in data import tool in ERPNext to periodically import to your Employee Checkin Document
```
Comment (michellealva) [inline erpnext_com/www/docs/user/manual/en/setting-up/articles/integrating-erpnext-with-biometric-attendance-devices.md]: The simplest possible solution (in terms of implementation complexity) would be to generate an Excel/CSV of the Check-in/Check-out and use the built-in data import tool in ERPNext to periodically import to your Employee Checkin Document

File: erpnext_com/www/docs/user/manual/en/setting-up/articles/integrating-erpnext-with-biometric-attendance-devices.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+    - Please refer to ERPNext's Documentation on Data Import Tool for more on how to do this.
```
Comment (michellealva) [inline erpnext_com/www/docs/user/manual/en/setting-up/articles/integrating-erpnext-with-biometric-attendance-devices.md]: a link for Data Import docs would be better

File: erpnext_com/www/docs/user/manual/en/setting-up/articles/integrating-erpnext-with-biometric-attendance-devices.md [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+
+1. API Integration:
+    - You can automate the process of importing the Biometric Punch Logs but integrating with the available API in ERPNext.
```
Comment (michellealva) [inline erpnext_com/www/docs/user/manual/en/setting-up/articles/integrating-erpnext-with-biometric-attendance-devices.md]: You can automate the process of importing the Biometric Punch Logs by integrating it with the available API in ERPNext.

File: erpnext_com/www/docs/user/manual/en/setting-up/articles/integrating-erpnext-with-biometric-attendance-devices.md [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+    - This API can be accessed at: `/api/method/erpnext.hr.doctype.employee_checkin.employee_checkin.add_log_based_on_employee_field`
+    - This method requires technical knowledge and you should probably get in touch with your ERPNext implementor or Biometric system vendor.
+
+1. Setup a python script on your computer to integrate ZKTeco or similar devices:
```
Comment (michellealva) [inline erpnext_com/www/docs/user/manual/en/setting-up/articles/integrating-erpnext-with-biometric-attendance-devices.md]: Set up -verb
Setup - noun
Here it should be set up

File: erpnext_com/www/docs/user/manual/en/setting-up/contents.md [hunk 8]
```diff
@@ -108,6 +108,7 @@ You'll find the global configuration options here, for module-wise setup, visit
 1. [Using Custom Domain On ERPNext](/docs/user/manual/en/setting-up/articles/using-custom-domain-on-erpnext)
 1. [Setup Two Factor Authentication](/docs/user/manual/en/setting-up/articles/setup-two-factor-authentication)
```
Comment (michellealva) [inline erpnext_com/www/docs/user/manual/en/setting-up/contents.md]: Set up

File: erpnext_com/www/docs/user/manual/en/setting-up/articles/integrating-erpnext-with-biometric-attendance-devices.md (hunk 1 above)
Comment (karthikeyan5) [reply erpnext_com/www/docs/user/manual/en/setting-up/articles/integrating-erpnext-with-biometric-attendance-devices.md]: done... added comma and changed 'E' in employee to lower case...

File: erpnext_com/www/docs/user/manual/en/setting-up/articles/integrating-erpnext-with-biometric-attendance-devices.md (hunk 2 above)
Comment (karthikeyan5) [reply erpnext_com/www/docs/user/manual/en/setting-up/articles/integrating-erpnext-with-biometric-attendance-devices.md]: done and done

File: erpnext_com/www/docs/user/manual/en/setting-up/articles/integrating-erpnext-with-biometric-attendance-devices.md (hunk 3 above)
Comment (karthikeyan5) [reply erpnext_com/www/docs/user/manual/en/setting-up/articles/integrating-erpnext-with-biometric-attendance-devices.md]: added 'the'

File: erpnext_com/www/docs/user/manual/en/setting-up/articles/integrating-erpnext-with-biometric-attendance-devices.md (hunk 4 above)
Comment (karthikeyan5) [reply erpnext_com/www/docs/user/manual/en/setting-up/articles/integrating-erpnext-with-biometric-attendance-devices.md]: added space before open braces...

File: erpnext_com/www/docs/user/manual/en/setting-up/articles/integrating-erpnext-with-biometric-attendance-devices.md (hunk 6 above)
Comment (karthikeyan5) [reply erpnext_com/www/docs/user/manual/en/setting-up/articles/integrating-erpnext-with-biometric-attendance-devices.md]: done

File: erpnext_com/www/docs/user/manual/en/setting-up/articles/integrating-erpnext-with-biometric-attendance-devices.md (hunk 7 above)
Comment (karthikeyan5) [reply erpnext_com/www/docs/user/manual/en/setting-up/articles/integrating-erpnext-with-biometric-attendance-devices.md]: I see... done.

File: erpnext_com/www/docs/user/manual/en/setting-up/contents.md (hunk 8 above)
Comment (karthikeyan5) [reply erpnext_com/www/docs/user/manual/en/setting-up/contents.md]: lol... this was not some this I added... changes it anyways...

Comment (karthikeyan5) [reply erpnext_com/www/docs/user/manual/en/setting-up/contents.md]: Though I am never going to get the difference between "set up" and "setup" anytime soon... :(

File: erpnext_com/www/docs/user/manual/en/setting-up/articles/integrating-erpnext-with-biometric-attendance-devices.md (hunk 5 above)
Comment (karthikeyan5) [reply erpnext_com/www/docs/user/manual/en/setting-up/articles/integrating-erpnext-with-biometric-attendance-devices.md]: added the link

**Pull request merged**